style.css

```diff
@@ -227,3 +227,80 @@ body .is-layout-flow .wp-block-group {
 	padding-right: 4.5rem;
 	padding-left: 4.5rem;
 }
+
+body.page-template-homepage {
+	padding-left: 0px;
+	padding-right: 0px;
+}
+
+.page-template-homepage header {
+	padding-left: clamp(1.25rem,((2vw - 0.75rem) * 10) , 6.25rem);
+	padding-right: clamp(1.25rem,((2vw - 0.75rem) * 10) , 6.25rem);
+}
+
+.wp-block-sensei-lms-course-list button {
+	text-transform: uppercase;
+	font-family: var(--wp--preset--font-family--league-gothic);
+	width: 100%;
+}
+
+.wp-block-sensei-lms-course-list .wp-block-sensei-lms-course-categories {
+	font-family: var(--wp--preset--font-family--system);
+}
+
+.is-style-group-left-border {
+	border-width: 0 0 0 1px;
+}
+
+.course-font-color-tertiary, .course-font-color-tertiary a {
+	color: var(--wp--preset--color--tertiary);
+}
+
+.wp-block-sensei-lms-course-list .wp-block-post-template .wp-block-post-featured-image {
+	margin-bottom: 0px;
+}
+
+.wp-block-post-featured-image img {
+	border: 1px solid var(--wp--preset--color--primary);
+	border-radius: 4px;
+}
+
+.wp-block-sensei-lms-course-list .wp-block-post-featured-image img {
+	border: 1px solid var(--wp--preset--color--tertiary);
+}
+
+.wp-block-image {
+	margin: 0px;
+}
+
+.course-homepage-pattern-parent-group > .wp-block-group {
+	padding-left: clamp(20px, calc(1.25rem + ((1vw - 4.8px) * 11.1111)), 100px);
+	padding-right: clamp(20px, calc(1.25rem + ((1vw - 4.8px) * 11.1111)), 100px);
+}
+
+.is-style-course-image-border img {
+	border: 1px solid var(--wp--preset--color--tertiary);
+}
+
+.wp-block-sensei-lms-course-list .featured-image-wrapper .sensei-lms-featured-badge,
+.featured-course-wrapper__featured-image .course-list-featured-label__featured-image {
+	font-family: var(--wp--preset--font-family--system);
+	font-size: var(--wp--preset--font-size--3-x-small);
+	background-color: var(--wp--preset--color--primary);
+	text-transform: uppercase;
+	padding: 3px 6px;
+	line-height: 1rem;
+	border-radius: 2px;
+}
+
+.featured-course-wrapper__course-categories a {
+	font-family: var(--wp--preset--font-family--system);
+	line-height: 145%;
+	padding: 3px 6px;
+}
+
+.wp-block-post-author__avatar img {
+	border: 1px solid var(--wp--preset--color--tertiary);
+	border-radius: 4px;
+}
+}
```

Comment:
it his extra `}` correct?

Reply:
Do you need this CSS or can you put the settings in the blocks? If the settings are inline in the block then its easier for users to change them...

Reply:
@gabrielcaires thanks for noticing, I've removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/b87083c1c01f4c84fdced59affdfc65b028f13f9 :)

Reply:
@scruffian thanks! :) I've removed all the styles which we could apply inline from here in this update https://github.com/Automattic/sensei-theme/pull/44/commits/0ac98bb70032bc77b6b62697655cb1e851b378e1